lib/src/screens/buy/buy_sell_page.dart

```diff
@@ -312,8 +312,12 @@ class BuySellPage extends BasePage {
 
     reaction((_) => buySellViewModel.isReadyToTrade, (bool isReady) {
       if (isReady) {
+        if (buySellViewModel.skipIsReadyToTradeReaction) {
+          buySellViewModel.skipIsReadyToTradeReaction = false;
+          return;
+        }
         if (cryptoAmountController.text.isNotEmpty &&
-            cryptoAmountController.text != S.current.fetching) {
+            cryptoAmountController.text != S.current.fetching && !buySellViewModel.isBuyAction) {
```

Comment:
this is causing an issue when entering the crypto amount not the fiat amount
```suggestion
            cryptoAmountController.text != S.current.fetching) {
```

Reply:
@Serhii-Borodenko just for reference